server/room.ts

```diff
@@ -266,45 +186,51 @@ export class Room {
   getHostState() {
     return {
       video: this.video,
+      videoDuration: this.videoDuration,
       videoTS: this.videoTS,
       paused: this.paused,
     };
   }
 
-  async resetRoomVM() {
-    const id = this.vBrowser && this.vBrowser.id;
-    this.vBrowser = undefined;
-    this.roster.forEach((user, i) => {
-      this.roster[i].isController = false;
-    });
-    this.cmdHost(undefined, '');
-    if (id) {
-      try {
-        await resetVM(id);
-      } catch (e) {
-        console.error(e);
-      }
+  cmdHost(
+    socket: Socket,
+    data?: PlaylistVideo,
```

Comment:
can this continue to take a string, and callers extract the url from playlistvideo if it came off the queue?